src/core.js

```diff
@@ -220,20 +220,6 @@ jQuery.extend( {
 		return obj != null && obj === obj.window;
 	},
 
-	isNumeric: function( obj ) {
```

Comment:
This doesn't belong in this PR, it should only be about `jQuery.now` deprecation.